admin/class-yoast-form.php

```diff
@@ -430,8 +431,9 @@ public function hidden( $var, $id = '' ) {
 	 * @param string $var            The variable within the option to create the select for.
 	 * @param string $label          The label to show for the variable.
 	 * @param array  $select_options The select options to choose from.
+	 * @param string $styled         Whether the select is styled. Default is unstyled.
```

Comment:
Based on the param is used and described I'd expect it to be a boolean, is there a reason you decided for a string? I think it would make the code more readable.

Reply:
Opted for a string because of:

**Self-Explanatory Flag Values for Function Arguments**
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#self-explanatory-flag-values-for-function-arguments

In a few words, would you be able to understand at first sight what the various params are in a call like the following?

`my_function( $something, true, true, false, true )`

Self-explanatory values solve the problem. I've updated the param description.